Sourcery/Models/TypeName.swift

```diff
@@ -54,13 +56,52 @@ final class TypeName: NSObject, AutoDiffable {
         }
     }
 
-    /// sourcery: skipEquality
-    /// sourcery: skipDescription
+    // sourcery: skipEquality
     var isVoid: Bool {
         return name == "Void" || name == "()"
     }
 
+    var isTuple: Bool {
+        if let actualTypeName = actualTypeName?.name {
+            return actualTypeName.isValidTupleName()
+        } else {
+            return name.isValidTupleName()
+        }
+    }
+
+    var tuple: TupleType?
+
     override var description: String {
-        return name
+        if let actualTypeName = actualTypeName {
```

Comment:
I don't think this should be doing anything like `aka`, this isn't debugDescription, the reason I've made it return `name` is because it's convienent to allow users to render it instead of having to reach through to name. So it should what you'd like to generate on callsite